lib/appsignal/integrations/redis.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Appsignal
+  module Integrations
+    module RedisIntegration
+      def process(commands, &block)
+        sanitized_commands = commands.map do |command, *args|
+          "#{command}#{" ?" * args.size}"
+        end.join("\n")
+
+        Appsignal.instrument "query.redis", id, sanitized_commands do
+          process_without_appsignal(commands, &block)
```

Comment:
```suggestion
          super
```

`process_without_appsignal` doesn't exist anymore.